app/config/FeatureSwitches.scala

```diff
@@ -52,6 +53,7 @@ class FeatureSwitchesImpl(featureSwitchConfig: Configuration) extends FeatureSwi
 
   def isReleasedInProduction(feature: String): Boolean = isConfigTrue(feature + ".released-in-production")
   def isPOARelevantAmountEnabled: Boolean = isEnabled("isPOARelevantAmount")
+  def isChargeReferencePoaAdjustmentChangesEnabled: Boolean = isEnabled("isChargeReferencePoaAdjustmentChanges")
```

Comment:
The name in application.conf is different so should be:
```suggestion
  def isChargeReferencePoaAdjustmentChangesEnabled: Boolean = isEnabled("chargeReferencePoaAdjustmentChanges")
```